include/NotificationsManager.php

```diff
@@ -213,8 +213,9 @@ private function formatNotifs($notifs, $ident = "") {
 				// Because we use different db tables for the notification query
 				// we have sometimes $it['unseen'] and sometimes $it['seen].
 				// So we will have to transform $it['unseen']
-				if($it['unseen'])
+				if (array_key_exists('unseen', $it)) {
```

Comment:
Out of curiosity, why not use the ubiquitous `isset()`, I don't remember there's a difference with ˋarray_key_exists()`?